dbfilter_from_header/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2013  Therp BV
+# © 2014  ACSONE SA/NV
+# Copyright 2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "dbfilter_from_header",
+    "summary": "Filter databases with HTTP headers",
+    "version": "13.0.1.0.0",
+    "website": "https://github.com/OCA/server-tools",
+    "author": "Therp BV, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "complexity": "normal",
```

Comment:
I think this key should be removed, it's not used neither by odoo nor by the OCA template.

Reply:
Removed. Thank you for your review :)